Vignette suggestions (#65)
b-cubed-eu/dubicube: vignettes/articles/group-level-sensitivity.Rmd

    $$
    \text{Error}_{s_j} = \hat{\theta}_{-s_j} - \hat{\theta}
    $$

    -   The **Relative Error** is the absolute error, normalized by the true estimate $\hat{\theta}$ and a small error term $\epsilon = 10^{-8}$ to avoid division by zero.

    $$
    \text{Rel. Error}_{s_j} = \frac{|\hat{\theta}_{-s_j} - \hat{\theta}|}{\hat{\theta} +\epsilon}
    $$

    -   The **Percent Error** is the relative error expressed as a percentage.

    $$
    \text{Perc. Error}_{s_j} = \text{Rel. Error}_{s_j} \times 100 \%
    $$

-   **Summary Measures**:

    -   The **Mean Relative Error (MRE)** is the average of the relative errors over all categories.

    $$
    \text{MRE} = \frac{1}{s} \sum_{j=1}^s \text{Rel. Error}_{s_j}
    $$

    -   The **Mean Squared Error (MSE)** is the average of the squared errors.

    $$
    \text{MSE} = \frac{1}{s} \sum_{j=1}^s (\text{Error}_{s_j})^2
    $$

    -   The **Root Mean Squared Error (RMSE)** is the square root of the MSE.

    $$
    \text{RMSE} = \sqrt{\text{MSE}}
    $$

## Getting started with dubicube

> The dubicube cross-validation method can be applied on any dataframe from which a statistic is calculated and for which a grouping variable is present.

Reviewer: ```suggestion
The **dubicube** cross-validation method can be applied on any dataframe from which a statistic is calculated and for which a grouping variable is present.
```

mean_obs(processed_cube$data)
```

> These values don’t tell us anything about reliability on their own.

Reviewer: ```suggestion
These values do not tell us anything about reliability on their own.
```

> Therefore, we perform a leave-one-species-out cross-validation (LOSO-CV) to investigate the impact of individual species being included in the calculation.

Reviewer: ```suggestion
Therefore, we perform a leave-one-species-out cross-validation (LOSO-CV) to investigate the impact of individual species being excluded from the calculation.
```

```

Indeed, looking at the individual error values reveals similar patterns for every year.

> The species with taxonkey 2481174 has an error of -2 in year 2011. This means that without this species the average number of occurrences per grid cell would be 2 untis lower than the estimate based on the full dataset (in this case around 46.8 instead of 48.8).

Reviewer: ```suggestion
The species with taxonkey 2481174 has an error of -2 in year 2011. This means that without this species the average number of occurrences per grid cell would be 2 units lower than the estimate based on the full dataset (in this case around 46.8 instead of 48.8).
```
